### PR TITLE
[SDK v1.4] ad -> banner, add new styles w/transparency

### PR DESCRIPTION
**Pull request by msub2**
finally ready to be reviewed

### CODE REVIEW

File: aframe/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@zestymarket/aframe-sdk",
-  "version": "1.3.0",
-  "description": "This is the A-Frame SDK for Zesty Ads integration.",
+  "version": "1.4.0",
+  "description": "This is the A-Frame SDK for Zesty Banners integration.",
```
Comment (limbofeather) [inline aframe/package.json:4]: singular?

**Review by limbofeather — APPROVED** [review]
Did you want to cut a new release after this?

**Comment by msub2** [comment]
Yes that was the intention

File: aframe/package.json (hunk 1 above)
Comment (msub2) [reply aframe/package.json:4]: probably yeah, will adjust

**Pull request merged**